### PR TITLE
Adding docs showing how to load scripts from a CDN

### PR DESCRIPTION
**Pull request by phillipskevin**
Closes https://github.com/stealjs/steal/issues/747.

### CODE REVIEW

**Comment by phillipskevin** [comment]
@matthewp let me know if any of the wording could be more correct. Also, I think you mentioned needing to set `server-development` and `server-production` for SSR to work, but I wasn't able to reproduce any issue with SSR without these.

File: docs/pages/loading-from-cdn.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Configure Steal to use the previously loaded file
+
+### Load Using a <script> Tag and Configure Steal to use Loaded Version
+
+```
+    <script src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
+    <script>
+      steal = {
+        instantiated: {
+          jquery: { 'default': window.jQuery, __useDefault: true }
```
Comment (matthewp) [inline docs/pages/loading-from-cdn.md]: Yuck!  I'd like to figure out how we can fix this so the user doesn't have to know about the nasty internals like this.

File: docs/pages/loading-from-cdn.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        }
+      };
+    </script>
+    <script src="./node_modules/steal/steal.js"></script>
+```
+
+### Prevent Steal from Looking for jQuery During Build
+
+In order for the production build to work, map jQuery to the `@empty` so Steal can find it during the build:
```
Comment (matthewp) [inline docs/pages/loading-from-cdn.md]: I would rephrase this slightly: `so Steal knows it doesn't need to try and find it.`

**Comment by matthewp** [comment]
How about also linking to these docs from the [Moving to Production](https://raw.githubusercontent.com/stealjs/steal/master/docs/pages/production.md) docs.

File: docs/pages/loading-from-cdn.md (hunk 1 above)
Comment (phillipskevin) [inline docs/pages/loading-from-cdn.md]: I agree. Should we create a separate issue to make something like `instantiated: { jquery: window.jQuery }` work?

**Comment by phillipskevin** [comment]
I added a link to the Moving to Production docs.

File: docs/pages/loading-from-cdn.md (hunk 1 above)
Comment (matthewp) [inline docs/pages/loading-from-cdn.md]: Yeah, there might already be one, ignore this comment as having anything to do with the doc changes.

Comment (pYr0x) [inline docs/pages/loading-from-cdn.md]: would this 
```jquery: { 'default': window.jQuery, __useDefault: true }```
let me use jquery in a module regular?

e.g.:
__foobar.js__
```
var $ = require('jquery');
$('body').append('hello');
```


also, do i have to set jquery as a dependency in package.json?

Comment (matthewp) [inline docs/pages/loading-from-cdn.md]: No, you don't need it as a dep.  `instantiated` puts it in the module registry.
